history/index.d.ts

```diff
@@ -1,18 +1,21 @@
-// Type definitions for history v2.0.0
-// Project: https://github.com/rackt/history
-// Definitions by: Sergey Buturlakin <https://github.com/sergey-buturlakin>, Nathan Brown <https://github.com/ngbrown>
+// Type definitions for history v3.0.0
+// Project: https://github.com/ReactTraining/history
+// Definitions by: Sergey Buturlakin <https://github.com/sergey-buturlakin>, Nathan Brown <https://github.com/ngbrown>, Viacheslav Slinko <https://github.com/vslinko>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-// types based on https://github.com/rackt/history/blob/master/docs/Terms.md
+// types based on https://github.com/ReactTraining/history/blob/master/docs/Glossary.md
 
-export as namespace History;
+export as namespace ReactTrainingHistory;
```

Comment:
The global case doesn't seem to work with the current definition. :(

```typescript
var ch = History.createHistory
```
doesn't resolve....